CONTRIBUTING.md

```diff
@@ -40,18 +40,18 @@ We love receiving contributions from our community. Any assistance you can provi
 Unsure of how to get started contributing to Red? Please take a look at the Issues section of this repo and sort by the following labels:
 
 * beginner - issues that can normally be fixed in just a few lines of code and maybe a test or two.
-* help-wanted - issues that are currently unassigned to anyone and may be a bit more involved/complex than issues tagged with beginner.
+* help-wanted - issues that are currently unassigned to anyone and maybe a bit more involved/complex than issues tagged with beginner.
```

Comment:
`maybe` and `may be` mustn't be confused with one another. One change you could make would be to add `are` before the `maybe`.

Reply:
Should i change it to "and are may be" or should i keep the maybe?

Reply:
You should keep at is easy as it was grammatically correct